app/Repositories/PostRepository.php

```diff
@@ -127,9 +127,11 @@ public function create(array $data, $signupId)
             $signup->save();
         }
 
-        // Edit the image if there is one
-        if (isset($data['file'])) {
-            $this->crop($data, $post->id);
+        if (! config('features.glide')) {
```

Comment:
Do we want to keep creating these, in case we decide to turn Glide off again?

Reply:
I think the flag should allow us to continue creating these.  If it is off, we crop, else we don't?

Reply:
I think this would mean we'd stop creating `edited_12313.jpg` crops once we turn on Glide, which would leave us with some 404ing images if we decided we had to turn it back off again after flipping the switch.

Reply:
Maybe a nice interim solution is to remove the `crop_rotate`, `crop_width`, etc. code paths from that function so we don't support custom crops anymore but still make the "default" edited crops?

Reply:
hmm I see, ok

Reply:
@DFurnes updated!

Reply:
@sbsmith86 It looks like you removed the pieces we don't need and just make the cropped image, but I thought the idea was to make that cropped image regardless of if glide is on or not, in case we have to turn it off?

Reply:
@katiecrane hmmm i don't think I understand the question, sorry! So this will still crop the image regardless of glide being on or off and store it in S3 as `edited_{post-id}.jpeg` but instead of using the crop values passed in the request, we just default to a 400x400 crop.

Reply:
@sbsmith86 hmm I'm seeing this inside `if (! config('features.glide'))` so it looks to me like this will only happen if glide is off. There are a lot of comments between that and the next line of code though, so I might be reading it wrong

Reply:
@sbsmith86 oh weird, I think I was somehow on an outdated version of the PR, nevermind!